Source/DafnyLanguageServer/Workspace/ChangeProcessors/Relocator.cs

```diff
@@ -50,14 +61,35 @@ CancellationToken cancellationToken
         return contentChanges.Aggregate(originalDiagnostics, MigrateDiagnostics);
       }
 
+      public List<Position> MigratePositions(List<Position> originalRanges) {
+        return contentChanges.Aggregate(originalRanges, MigratePositions);
+      }
+
       private IReadOnlyList<Diagnostic> MigrateDiagnostics(IReadOnlyList<Diagnostic> originalDiagnostics, TextDocumentContentChangeEvent change) {
         if (change.Range == null) {
-          throw new System.InvalidOperationException("the range of the change must not be null");
+          return new List<Diagnostic>();
```

Comment:
What makes `null` allowed now?

Reply:
This PR:;
https://github.com/dafny-lang/dafny/pull/1816